sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -430,6 +430,99 @@ test_suite(
     tests = [":e2e_bootstrap_entry_{}".format(lc_state.lower()) for lc_state in structs.to_dict(CONST.LCV)],
 )
 
+otp_json(
+    name = "otp_json_bootstrap_rma",
+    partitions = [
+        otp_partition(
+            name = "CREATOR_SW_CFG",
+            items = {
+                "CREATOR_SW_CFG_RMA_SPIN_EN": "0x739",  # HARDENED_BOOL_TRUE
+                "CREATOR_SW_CFG_RMA_SPIN_CYCLES": "10",
```

Comment:
TODO: Try larger numbers. This might be the cause of the flakiness I'm seeing.

Reply:
I'm a little puzzled — I did some estimates based on a 10 MHz CPU frequency and waiting 10-100 ms, but those new cycle counts made the test fail. How are 10 cycles enough time for OpenOCD to connect to lc_ctrl reliably? I just ran the test 30 times in a row without any failures.

Reply:
Interesting, I would expect longer wait durations to be more reliable. @tjaychen @msfschaffner any thoughts?

Reply:
Are you connecting to LC_CTRL or to RV_DM?

In any case, it takes a while until ROM_CTRL has finished checking the ROM and the ROM can actually execute, so the actual window during which you can connect is larger than just 10 cycles.

But I agree, making this wait count larger should actually make the test more reliable.

Reply:
I'm connecting to LC_CTRL in this test. Good point, that explains why 10 cycles is not too short.

I tried tuning it in terms of milliseconds earlier, but everything I did seemed to make connecting to the TAP flakier, or just outright fail. I suspect those numbers may have been too huge, so I'll give this another shot.

Reply:
Ah one question, in what lifecycle state are you booting the system?
And when do you drive the TAP selection straps?

Reply:
The lifecycle state is PROD. For the second question, I'll give you a little more info than you asked:

1. First, we load the bitstream.
2. We call the test's [reset()](https://github.com/lowRISC/opentitan/pull/17029/files#diff-d3301054e747c870c426cb35d9b9a97b5a9a01ec4c1d84dfcde78433cd785d4fR130) function. This drives the TAP selection straps, resets the chip, sleeps, and removes the strapping.
3. Finally, we connect OpenOCD and run the TCL test script.

Reply:
OK, one thing to note is that the pinmux does not sample the TAP straps continuously in PROD (in test-enabled states it however does that, see [this](https://docs.opentitan.org/hw/ip/pinmux/doc/#strap-sampling-and-tap-isolation) description). Hence, if you do not apply the straps right after reset, and wait too long, you won't be able to select the LC TAP anymore in PROD.